Dockerfile

```diff
@@ -13,6 +13,10 @@ ARG PIP_OPTIONS=.
 #     desired-packages \
 #     && rm -rf /var/lib/apt/lists/*
 
+# Useful for checking if port for PV service is correct
+# RUN apt-get update
```

Comment:
Are these `Run` statements ment to be commented out?

Reply:
Oops, I meant to remove them